### ISSUE TITLE
"To:" header in web ui 

### ISSUE THREAD

**Issue by tommywo** [jhillyerd/inbucket#34]
now, to get mails from friend@here.com and not from friend@nothere.com
user have to open friend inbox and open each message source to check To header
It would be easier if there would be "To" header display in web ui
![screen shot 2016-03-23 at 12 00 34](https://cloud.githubusercontent.com/assets/11270518/13983265/eb97fa56-f0ee-11e5-872e-a06cd0454394.png)

**Comment by jhillyerd** [comment]
It took me a second to see it, but it's there.  I guess it should be moved on top of the Date though, current ordering doesn't make sense.

**Comment by jhillyerd** [comment]
Implemented in #39

**Comment by jhillyerd** [comment]
Ok, I've checked in a change that parses the content of to/from headers, and stores To as a slice.  It looks like Go likes to render mail.Address entities with surrounding angle brackets, even if not needed.  Kinda weird, but at least it makes things consistent...?
```
  {
    "mailbox": "moo",
    "id": "20160921T215907-0001",
    "from": "<james@archbox.localdomain>",
    "to": [
      "<moo@inbucket.local>",
      "\"Alt User\" <alternate@inbucket.local>"
    ],
    "subject": "Swaks Multi-Recipient",
    "date": "2016-09-21T21:59:07.196283804-07:00",
    "size": 436
  },
```

Note:  I did a rebase & force-push on the feature/to-header branch, so you might have some problems fetching/merging it if you already have it checked out.

**Comment by tommywo** [comment]
It's unusual, but still RFC 5322 compliant, so I don't mind.
I will create a pull request to address this behavior in Go.

**Comment by jhillyerd** [comment]
Merged in 145e71dc438e3f0ccb51697e9f33513b32c7e8a2
